### PR TITLE
Add `disableAnimations` option

### LINKED ISSUE #10: Disable animations?
There's no point in running animations when you just want a screenshot.

We could inject some CSS overrides like this: https://dev.webonomic.nl/how-to-disable-css-transforms-transistions-and-animations

Thoughts?

### PR DESCRIPTION
**Pull request by hicom150**
You can disable all transformations, animations and transitions at once, or you can choose individually.

Fixes #10

### CODE REVIEW

**Comment by sindresorhus** [comment]
I don't think we should touch `transform` at all. It would break way to many sites and it's not very useful to disable it.

**Comment by sindresorhus** [comment]
That leaves `animation` and `transition`. I don't see the need to toggle them individually. I would prefer just one `disableAnimations: true/false` option.

File: index.js [hunk 1]
```diff
@@ -22,6 +22,23 @@ const removeElements = elements => {
...
 
+const disableAnimations = options => {
+	const defaults = {transform: false, animation: true, transition: true};
+	options = {...defaults, ...options};
+
+	const transform = options.transform ? 'transform: initial !important;' : '';
+	const animation = options.animation ? 'animation: initial !important;' : '';
+	const transition = options.transition ? 'transition: initial !important;' : '';
+
+	const rule = `*,:after,:before{${transform}${animation}${transition}}`;
```
Comment (sindresorhus) [inline index.js]:
```suggestion
	const rule = `*,::after,::before{${transform}${animation}${transition}}`;
```

File: index.js [hunk 2]
```diff
@@ -22,6 +22,23 @@ const removeElements = elements => {
...
+
+	const style = document.createElement('style'); // eslint-disable-line no-undef
+	document.body.append(style); // eslint-disable-line no-undef
+
+	const {sheet} = style;
+	sheet.insertRule(rule);
```
Comment (sindresorhus) [inline index.js]:
```suggestion
	style.sheet.insertRule(rule);
```

You don't need `sheet` for anything else, so no point in a variable for it.

**Comment by sindresorhus** [comment]
Can you also add the option to https://github.com/sindresorhus/capture-website-cli?

**Comment by sindresorhus** [comment]
And you need to add the option to https://github.com/sindresorhus/capture-website/blob/master/index.d.ts

**Comment by sindresorhus** [comment]
@hicom150 Ping :)

**Comment by hicom150** [comment]
First of all, thank you @sindresorhus for taking the time to make all the suggestions 😉 and sorry for the late late response 😅
I think I've applied all the suggestions, but if you find anything else I'll be ready to change 💪

**Pull request merged**